projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/models/query-list/query-list-item/query-list-item.model.ts

```diff
@@ -50,6 +50,11 @@ export class QueryListItem implements CaseMessage {
   }
 
   public get responseStatus(): QueryItemResponseStatus {
-    return this.children?.length > 0 ? QueryItemResponseStatus.RESPONDED : QueryItemResponseStatus.NEW;
+    if (this.children?.length > 0) {
+      return this.children.length % 2 === 1
```

Comment:
Can we find a more robust way of determining if this is responded or new, for example using the associated user Id, or the status of the message or perhaps a change to the data?